pubsub/aws/snssqs/snssqs.go

```diff
@@ -51,7 +51,16 @@ type snsSqsMetadata struct {
 	SessionToken string
 	// aws region in which SNS/SQS should create resources.
 	Region string
-
+	// name of the queue for this application. The is provided by the runtime as "consumerID".
+	sqsQueueName string
+	// name of the dead letter queue for this application.
+	sqsDeadLettersQueueName string
+	// flag to SNS and SQS FIFO.
+	fifo bool
+	// a namespace for SNS SQS FIFO to order messages within that group. limits consumer concurrency if set but guarantees that all
+	// published messages would be ordered by their arrival time to SQS.
+	// see: https://aws.amazon.com/blogs/compute/solving-complex-ordering-challenges-with-amazon-sqs-fifo-queues/
```

Comment:
This would probably be useful information to include in https://github.com/dapr/docs/issues/1977 as well

Reply:
ok. do you suggest leaving this and other (quite verbose) comments or move them into the official dapr docs?

Reply:
You can leave the comments here and add them to docs.